### ISSUE TITLE
Can't run converted model- Op (Slice) [ShapeInferenceError] 'axes' has duplicates

### ISSUE THREAD

**Issue by israelwei** [onnx/tensorflow-onnx#986]
**Describe the bug**
I converted a yolo3 model from Tensorflow (1.15) to ONNX, using the flag --opset=11.
The model is converted and saved, but whenever I parse the model (using onnx-runtime for example) I get the error of:
 Op (Slice) [ShapeInferenceError] 'axes' has duplicates
This happens due to some stridesSlice node. This node indeed has duplicates in the "begin/end" axes in the original pb file ([0, 0], and also in the onnx file in "axes".
These strides are not an issue in Tensorflow and it's possible to apply inference correctly on the model in Tensorflow. 
It's worth mentioning that there's negative stride in this node (this are other nodes with this structure).
Also, according to the code, it seems this node corresponds to the following command of slicing:
grid_shape[::-1]  
Meaning It's probably related to "reverse" slicing which counts as dynamic behavior for shape inference.
the node's details in the file pb:
![image](https://user-images.githubusercontent.com/23454156/85948777-4b9d8b80-b95b-11ea-89c4-19cf61bd3b7c.png)
the node's details in the file onnx file:
![image](https://user-images.githubusercontent.com/23454156/85948890-16456d80-b95c-11ea-9387-462d1f6e74c5.png)
It's noteable that the "starts"/"ends" for this node in the onnx file is a bit weird, as the values there are very large.
**Urgency**
This is quite urgent.

**System information**
- OS Platform and Distribution: Linux Ubuntu 18.04.3
- Tensorflow Version: 1.15
- Python version: 3.6

**To Reproduce**
I can't share the model unfortunately.

**Expected behavior**
I expected the onnx inference to be run without a shape inference error.
**Screenshots**
Attached above.

**Additional context**
This is a yolo3 model we adjusted to our needs, trained on Keras, applied freeze on the model to save it as pb file. Then I converted the pb file to onnx using tf2onnx, with opset as 11. 

Any help why this happens and how it can be fixed will be much appreciated.

**Comment by jignparm** [comment]
The high values for starts and ends is fine. The axes=[0,0] does not seem valid -- each axis value needs to be unique. 

The axes list  at this location is somehow getting the same axis value appended twice: https://github.com/onnx/tensorflow-onnx/blob/master/tf2onnx/onnx_opset/tensor.py#L689

Since you cannot share the model, is it possible to construct a small test model that exhibits this error?

**Comment by israelwei** [comment]
The way to reproduce this is by creating a node with "reversed slicing", i.e. grid_shape[::-1]
When I edited the graph and converted this node into a node of the format of tf.concat[second_shape_value, first_shape_value] , I could apply inference on the onnx model without errors.
This kind of slicing makes the same axis twice also in the pb model- though it's still possible to apply inference correctly in tensorflow with this. Only on onnx this causes issues, so a possible workaround is to do what I suggested here...
